chapters/04/dbdiagram.dbml

```diff
@@ -12,7 +12,7 @@ Table dim_customer {
   total_amount float
 }
 
-Table fact_orders {
+Table rders {
```

Comment:
Typo here. Btw are you removing fact/dim so it's more user friendly?

Reply:
Yes, it is based on comment from Jarda 
> [zde](https://strvcom.github.io/dbt-workshop/chapters/04/exercise.html) v prvním kroku po nás autor chce, abychom vytvořili dimenzi zákazníku, ale podle popisku tabulka bude spíše faktovka než dimenze (ke každému zákazníkovi se napočítávají agregované hodnoty, které se mohou v čase hodně měnit)
We have discussed it and I think he is quite right and according to the theory the dimension table we were creating was actually not a valid dim table. I have also checked the official dbt docs how they defined it (since it is based on jaffle shop data - https://docs.getdbt.com/guides/best-practices/how-we-structure/4-marts) and they are also using just customers/orders tables and not mentioning any dimensional modelling. That's why I would skip it also, because it is not correct and moreover not important for this workshop.